lib/svg.dart

```diff
@@ -29,29 +29,13 @@ final Svg svg = Svg._();
 class Svg {
   Svg._();
 
-  /// A global override flag for [SvgPicture.cacheColorFilter].
-  ///
-  /// If this is null, the value in [SvgPicture.cacheColorFilter] is used. If it
-  /// is not null, it will override that value.
-  @Deprecated('This no longer does anything.')
-  bool? cacheColorFilterOverride;
```

Comment:
Please don't remove these. There are usages of this API in some customers and removing them makes it much harder to migrate those customers to the new version.

Reply:
Especially with the doc comment still pointing towards this "doing something" this is quite confusing.